<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="registered_image">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="green_purple">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_menu">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Zpět do hlavního menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_result">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Uložit registrovaný obraz</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
